app/src/main/java/io/pumpkinz/pumpkinreader/data/NewsAdapter.java

```diff
@@ -80,15 +81,19 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
                 News news = this.dataset.get(position);
                 NewsViewHolder newsViewHolder = (NewsViewHolder) viewHolder;
 
-                int color;
-
                 if (PreferencesUtil.isNewsRead(fragment.getActivity(), news)) {
-                    color = fragment.getResources().getColor(R.color.grey_200);
+                    TypedValue dimTitleColor = new TypedValue();
```

Comment:
I think we can save a few lines by extracting the
```
newsViewHolder.getTitle().setTextColor(titleColor.data);
newsViewHolder.getScore().setTextColor(titleColor.data);
```
The `if` part only determines whether `titleColor` is from `R.attr.dim_title_color` or `R.attr.title_color`.